packages/tangle/images/TestBookerMarkerMappings.html

```diff
@@ -5,7 +5,7 @@
nd/jyy+lhzkiN9RywTdpJRigBjkTLwba8M2/t4RY/sSRd3/b9TPxaWi01yfdJFXRMmefuR1Rfm9AgTxdKlQYQs0Nf2dZXtU0rmpIpaHOOAkOWLnGG2KBeI8LvAb6gnSmlfOOV1jKdC0azNYrCh+RPclP+fjPT6zKhHarQhAZhxhLMnQXdjAXtQPJIWxY0iO2Bb8FaZymrKxrR1NFqO4HNXDsWwJDlR0zRb23YnZXq2mx/hZeFv7S6Yslp/g+qu2vzONirUyPLjGz0QLdmG5wCxJg74YJ2Oqkcf7MfH3r1lhqy9JtqpcWr0wQ622wjneBeivXk4TF9r+PBayDrJsKvP0hHFxFGlkomPP7929f7559Pfweffj9/+Y9+6Q8Cro8ZmrO61QBWsD1UibYEFZWLvChs81VVqn5nYbVdztfDyQ8+YfMyK+gsn82TL1s7wcSZuq3RiyjP5Jn7ZKF6M3MjcbEgipBGkUfFsmIJyywk9jxZaS+zeitvFeeStIeuY1MIYXLOlrt/yqt4UJ7JKPfdwJqn5Nfv+t8imMUh6mHdYHYFWRpC0m4PyeRlFIbx+gKbtOn9TTgM0jP+Dw==&lt;/diagram&gt;&lt;diagram name=\&quot;UTXO DAG\&quot; id=\&quot;zJlAbjRBJoZRiOzCOWW2\&quot;&gt;7Z1be9o4EIZ/DZfZR5Z8vAyn0GzTZEu3TXPTx9gC3Bgra5QE+uvXgAzGEg4HGxkMvWgsH2Lrnfk0Gg9KDTVGk5vQfhneERf7NQjcSQ01axAqKtSj/2Yt00WLoWuLhkHoueygVUPX+4NZI2Ctr56Lx2sHUkJ86r2sNzokCLBD19rsMCTv64f1ib/+W1/sAeYauo7t860/PJcOF60QIX21o4O9wTD+1bqmLvaM7Pho9ijjoe2S90QTatVQIySELn4aTRrYn/Ve3DFNlTR6dNiHT9/aX8nobtTXr64WF2vvcsryGUIc0L0v/Q8deth4ct50gHTDff73Xo1PAW+2/8o6rB7agTOM2hrsqek07suQvAYunl1OqaH6+9CjuPtiO7O975H1RG1DOvLZ7r7n+w3ik3B+LnJtbPadqH1MQ/KME3t0x8S9frTHDh1mPyDaesMh9SKO1743CKK2HqGUjKIdW3YG67TZZfAkYQusc24wGWEaTqND2F495swsHcXb7yu7QYC1DRMmA1XWaDNbHSyvvcIR/cCIiOn07u47AeoozSaCf77f9kMEH5fPIKDzKVc6NgQAakI6bRB9tqfTJwFNnN6ffwqkZhjaOjWdp6YgjaemaDlQe5vYN9pLvU6s1mPr19PV52vcvUKbqXVypQaAac/ZcNQA0IwyU9MR2IKawNdyoSZ8BriZ2nWu1LDiatgQUbN0A9m6fCVE5jqdZacn6agiOnpRdIzNdJq50ukBjLAu9ClsAtOUT0cBcE88CBWFx9yMp5Urnn4f644wjHANq7eL5BWFx9RL5zwWR6dioZ0B9mViFMVEyYi86/ni0bDpqiI8JuwhvQTjjaaWzmUUPvS+wQEee+PD2KT7lpIXgchps39Ch5p/+IBt8eFIzo7qsjvddhJ5cLSgCuAponmTWRg8Xt+wO8BxT5CQDsmABLbfWrXWVxhnPbc65jOZIZrD+40pnbLutV8pWUeLJx59nJ3+l6GxzZ+JXc0Ju/R8YxpvBNEDL85SFSNu+JncuzpxvrV25gMOvajPcMgad2M8Jq+hg7OOY6kQaocDnHVByCZYs27OtJkQ+zb13taTMyIDYKc+EC96ktW8IS3k6an34k7ZWSkzWt7GAZaF5FoW3MuytF3sSpoJmXmb0GEaonIDwHUNRTNBoM1sEHy6/5Y1LQS7R7b7i35Ryq6mQllDoOxQIOx55MPET6CdpPuZqHTCbpRN2NNz2mMLu865uwxTS3k4ACz5uldsIctmkFouJRfkjqqm5Ol8q3wlN09SySUGUjGejyUblsv9LLmk5Qnm1sSQUSpikE9NNaommNYWoa8op1GYYMa0ZbnR6ekl3FYvUbm8D3Le9y20g7HtUI8EgtTWTp6Xzj+OPNedmwr3nqUPN7xn0Xu6phfoeqaRmgqI3g0fNViB/Bv9TtUEMU1FmOY9LhX1ooi7KaK2rSLq5VJELUsRz18QNa10gshnS1oLQVRAZRQxjaUEimhcFHE3RTS3VESklEsR+XqbZIx49oqIrLIpYmwgyXqBqoWIaSpHFcTga+PWal9//zEYj5q37hMJBu9Xkt/WnpweIsGcWdix0lKMwoJrPgS8YN4Vs7BjyzXqxbed0Nh21TTWUMoWdKKLyO7ofdtWxCBpKpt53+Kgk7eCcws606531KDzfjr6bX59+McY9ibNh1/PVue5e3UyCbC8PUiQyBJ2ULky+4jPYzWrNn4pAJVuANNPxY3KMoBtXQiUe/HYYaD5QqDEAMaXFp7bAMb5nvwRjM+afKq8IqpAk0zlMp/eKIhZYcaH4Yi06XTWXW/IIVdQDhE8ouPhRsv78/hk/TftXjc7P760zb4lvSqyzJ4n7DDBREB4nCXL88SYJZdEHmPidhAuRVoFedZtJ5Tyc+WiFJj6lucxCyKFUPgShOX3p28yaOyx5ACwEbBFNOpN45C1cZbf7igMGpL6xWohtoyFPNq5YsPIMcWL49SBpZUamyJ1/RUhtktgsttIZ2070oFSjXT8SiGJOQGfbj67KYEis7Ak00ASSP6uXPCRxnLUpLEYC+Sw3FYOC5JZASnGcnkZvds4JVqeIdPgSzJQCZZnSGav+Fc6ZzdUpZ3vqEOVsCiMH6nuqiaJRmqgUkWROxCsRloYlMvXBjcKYlZl44cVkNLi9qy73qCGfLrx3NQw7XdImJ4qyO+EpZO8GH6pmhjq6vrSzEcVQ+F62uBUxHB/Ucuq4/2w3leaqGXdtVjUzl/T0u5TpKZFm6u/Y7BYU2n15yBQ638=&lt;/diagram&gt;&lt;/mxfile&gt;&quot;}"></div>
```

Comment:
We still have an obsolete MD version of this. Let's delete it if we decided not to update it :sunglasses: